app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    sepp_command = command.split()
```

Comment:
The command string should be checked to ensure it contains exactly three elements and starts with 'mv'. This is a requirement according to the checklist.